pkg/v1/mutate/mutate.go

```diff
@@ -113,6 +113,29 @@ func Config(base v1.Image, cfg v1.Config) (v1.Image, error) {
 	return ConfigFile(base, cf)
 }
 
+// IndexAnnotations mutates the provided v1.Image to have the provided annotations
+func IndexAnnotations(base v1.Image, annotations map[string]string) (v1.Image, error) {
```

Comment:
Sorry, my comment wasn't clear, just call this `Annotations`

Reply:
done.